install/windows.md

```diff
@@ -0,0 +1,76 @@
+permalink: /docs/install/windows/
+---
+This page describes how to install Airsonic under Windows as Stand-alone WAR, without using Tomcat. It also contains information about how to setup Airsonic as a Windows Service and let it run under an user/service account.
+
+> **NOTE**: If you like to run Airsonic by using HTTPS, take a look at the proxy section about [IIS](/docs/proxy/iis/).
+
+##### Prerequisites
+
+Download the latest [Airsonic Release](https://github.com/airsonic/airsonic/releases), you need the `airsonic.war`.
+
+Install [OpenJDK](https://openjdk.java.net) or another Java implementation. If you're using vanilla Airsonic, make sure you use version 13 or below. For Airsonic Advanced with 11.0.0 or newer you need to have at least version 11, but 14 is fine as well. Make sure to add Java to your `PATH` variable.
+
+If you like to run Airsonic as a Windows Service, download the [Non-Sucking Service Manager (NSSM)](https://nssm.cc/download).
+
+##### Install Airsonic
+
+Before we start, doublecheck if Java is working as expected and you setup the `PATH` environment variable right. Open a command prompt or the PowerShell console and type `java -version`. This should print the installed Java version.
+
+Create a new directory `C:\Program Files\Airsonic` and move the `airsonic.war` to this folder.
+
+Create an `airsonic.cmd` file in this folder and add the following content.
+
+```
+java.exe -Dairsonic.home="C:\\Program Files\\Airsonic" -Dserver.port=4040 -jar airsonic.war
+```
+
+Now you're ready! Start the `airsonic.cmd` file by double clicking on it, wait some seconds and browse to `http://localhost:4040`. You should see the Airsonic logon page.
+
+##### Installing Airsonic as Windows Service
+
+As it might be a bit inconvenient to start Airsonic after every restart of your computer/server by yourself, you can setup a Windows Service. This Windows Service will start Airsonic automatically when you boot your computer, you don't even need to logon.
+
+> **NOTE**: To run a Java application as a Windows Service there would be a [Java Service Wrapper](https://wrapper.tanukisoftware.com), but it's not really easy to setup. That's why we go for NSSM.
+
+Create a new directory `C:\Program Files\NSSM` and copy `nssm.exe` to this folder.
+
+Open a command prompt or the PowerShell console and change the directory to the NSSM folder `cd C:\Program Files\NSSM`.
+
+Type `nssm install Airsonic` to open the NSSM service installer and start setting up the Windows Service for Airsonic.
+
+Set the following settings in the `Application` tab.
+
+- Path = `java.exe`
+- Startup directory = `C:\Program Files\Airsonic`
+- Arguments = `-Dairsonic.home="C:\\Program Files\\Airsonic" -Dserver.port=4040`
+
+Open the `Shutdown` tab and remove all the flags in the checkboxes excepted the one for `Generate Contral-C`. Set the timeout to `300000`.
+
+> **NOTE**: This very long timeout (5 minutes) is needed, as Airsonic takes some time to shutdown properly.
```

Comment:
(off-topic) wondering why Airsonic might take 5 minutes to shut down? I don't run Windows, but on my machine Airsonic never takes more than a few seconds at most to shut down?

Reply:
Seems to be a bug I think, it really takes a very long time. The log always says "2020-05-03 16:05:50.223  INFO --- o.a.player.service.UPnPService           : Disabling UPnP/DLNA media server" and the airsonic.lck (HSQLDB) file is removed after some seconds, but the java.exe is still running for some more minutes...
I'll try to do more investigation about this and create an issue (or a PR) if I know more about it.
For now I'll let the manual is it is, if that's fine.

Reply:
Seems to be related to DLNA. After disabling DLNA, Airsonic respectifly the java.exe closes within seconds. I'm going to update the manual and let this part about the timeout away.

Reply:
Uh, Cling (the library Airsonic uses for DLNA) acting up again... :disappointed: